README.md

```diff
@@ -166,39 +167,59 @@ for details of what is already available when running GitHub Actions.
+| .travis.helm_install.yml                  | Preinstalled                                                                  |
+| .travis.home_bin_path.yml                 | Not yet determined                                                            |
+| .travis.java.yml                          | See [Java Setup section](#setup-maven-build-options)                          |
+| .travis.java_config.yml                   | See [Java Setup section](#java-setup)                                         |
+| .travis.java_docker.yml                   | See [Java Setup section](#setup-maven-build-options)                          |
+| .travis.jq_install.yml                    | Preinstalled                                                                  |
+| .travis.kcadm_install.yml                 | Not yet determined                                                            |
+| .travis.kubepug_install.yml               | [setup-kubepug](.github/actions/setup-kubepug/action.yml)                     |
+| .travis.kubernetes_install.yml            | Preinstalled                                                                  |
+| .travis.maven_config.yml                  | See [Java Setup section](#java-setup)                                         |
+| .travis.pre-commit.yml                    | [pre-commit](.github/actions/pre-commit)                                      |
+| .travis.rancher_cli_config.yml            | [setup-rancher-cli](.github/actions/setup-rancher-cli/action.yml)             |
+| .travis.rancher_cli_install.yml           | [setup-rancher-cli](.github/actions/setup-rancher-cli/action.yml)             |
+| .travis.rancher_cli_kubernetes_config.yml | [setup-rancher-cli](.github/actions/setup-rancher-cli/action.yml)             |
+| .travis.srcclr_install.yml                | Not yet determined                                                            |
+| .travis.terraform-docs_install.yml        | [setup-terraform-docs](.github/actions/setup-terraform-docs/action.yml)       |
+| .travis.terraform_install.yml             | Preinstalled                                                                  |
+| .travis.tflint_install.yml                | Not yet determined                                                            |
+| .travis.trigger.yml                       | Not yet determined                                                            |
+| .travis.veracode.yml                      | [veracode](.github/actions/veracode)                                          |
+| .travis.yq_install.yml                    | Preinstalled                                                                  |
+
+### Default environment variables
+
+| Travis CI           | GitHub Actions          |
+|---------------------|-------------------------|
+| ${TRAVIS_BRANCH}    | ${{ github.ref_name }}  |
+| ${TRAVIS_BUILD_DIR} | ${{ github.workspace }} |
+| ${TRAVIS_COMMIT}    | ${{ github.sha }}       |
+
+### Make build logs verbose again
```

Comment:
```suggestion
### Get back maven build output
```
I think this is specific to maven builds, and this is not strictly a verbosity issue: we just didn't have any output with Travis (except errors)